HTTPlayground/settings.py

```diff
@@ -1,13 +1,25 @@
 from pathlib import Path
 from sys import argv
+from jinja2 import Environment, FileSystemLoader
 
-PORT = 8000
-URL = 'localhost'
+SERVERS_INFO = {
+    'URL0': '127.0.0.1', 'PORT0': 8000,
+    'URL1': '0.0.0.0', 'PORT1': 8080  # 127.255.255.254
+}
 
-if len(argv) > 2:
-    PORT = int(argv[2])
-    URL = argv[1]
+IS_MAIN_SERVER = not int(argv[1])
+
+PORT = SERVERS_INFO.get(f'PORT{argv[1]}', SERVERS_INFO['PORT0'])
+URL = SERVERS_INFO.get(f'URL{argv[1]}', SERVERS_INFO['URL0'])
+
+ACCESSORY_PORT = SERVERS_INFO.get(f"PORT{1 - int(argv[1])}", "PORT1")
```

Comment:
I think you might want `ACCESSORY_*` creds to be `None` for the second server, not the ones of the current server itself.

Reply:
In the response template of the second server the url of the main server is used (in the "Home" button).

> I think you might want `ACCESSORY_*` creds to be `None` for the second server, not the ones of the current server itself.

Reply:
OK.